### ISSUE TITLE
Error when the project is generated

### ISSUE THREAD

**Issue by lucianboboc** [IBM-Swift/generator-swiftserver#190]
I have macOS Sierra, Xcode 8.3.2 and Swift 3.1.
I used `yo swiftserver` to generate a new project and most of the packages were fetched.
But after this i got this error:

```Fetching https://github.com/IBM-Swift/Kitura.git
Fetching https://github.com/IBM-Swift/HeliumLogger.git
Fetching https://github.com/IBM-Swift/CloudConfiguration.git
Fetching https://github.com/RuntimeTools/SwiftMetrics.git
Fetching https://github.com/IBM-Swift/Kitura-net.git
Fetching https://github.com/IBM-Swift/SwiftyJSON.git
Fetching https://github.com/IBM-Swift/Kitura-TemplateEngine.git
Fetching https://github.com/IBM-Swift/LoggerAPI.git
Fetching https://github.com/IBM-Swift/BlueSocket.git
Fetching https://github.com/IBM-Swift/CCurl.git
Fetching https://github.com/IBM-Swift/BlueSSLService.git
Fetching https://github.com/IBM-Swift/Swift-cfenv.git
Fetching https://github.com/IBM-Swift/Configuration.git
Fetching https://github.com/RuntimeTools/omr-agentcore.git
Fetching https://github.com/IBM-Swift/Kitura-WebSocket.git
Fetching https://github.com/IBM-Swift/Kitura-Request.git
error: fatal error: Error raised at top level: Error Domain=NSCocoaErrorDomain Code=4 "“org.eclipse.paho.mqtt.c” couldn’t be moved to “src” because either the former doesn't exist, or the folder containing the latter doesn't exist." UserInfo={NSSourceFilePathErrorKey=/Users/lucianboboc/Desktop/kituraApp/org.eclipse.paho.mqtt.c, NSUserStringVariant=(
    Move
), NSDestinationFilePath=/Users/lucianboboc/Desktop/kituraApp/src/paho, NSFilePath=/Users/lucianboboc/Desktop/kituraApp/org.eclipse.paho.mqtt.c, NSUnderlyingError=0x7f853b4252c0 {Error Domain=NSPOSIXErrorDomain Code=2 "No such file or directory"}}: file /Library/Caches/com.apple.xbs/Sources/swiftlang/swiftlang-802.0.53/src/swift/stdlib/public/core/ErrorType.swift, line 188
Current stack trace:
0    libswiftCore.dylib                 0x00000001133da130 swift_reportError + 129
1    libswiftCore.dylib                 0x00000001133f6a40 _swift_stdlib_reportFatalErrorInFile + 100
2    libswiftCore.dylib                 0x00000001131e7ba0 (_assertionFailure(StaticString, String, file : StaticString, line : UInt, flags : UInt32) -> Never).(closure #1).(closure #1).(closure #1) + 124
3    libswiftCore.dylib                 0x000000011339e360 partial apply for (_assertionFailure(StaticString, String, file : StaticString, line : UInt, flags : UInt32) -> Never).(closure #1).(closure #1).(closure #1) + 93
4    libswiftCore.dylib                 0x00000001131e7250 specialized specialized StaticString.withUTF8Buffer<A> ((UnsafeBufferPointer<UInt8>) -> A) -> A + 342
5    libswiftCore.dylib                 0x00000001133a0880 partial apply for (_assertionFailure(StaticString, String, file : StaticString, line : UInt, flags : UInt32) -> Never).(closure #1).(closure #1) + 144
6    libswiftCore.dylib                 0x00000001131e7760 specialized specialized String._withUnsafeBufferPointerToUTF8<A> ((UnsafeBufferPointer<UInt8>) throws -> A) throws -> A + 784
7    libswiftCore.dylib                 0x0000000113361f40 partial apply for (_assertionFailure(StaticString, String, file : StaticString, line : UInt, flags : UInt32) -> Never).(closure #1) + 185
8    libswiftCore.dylib                 0x00000001131e7250 specialized specialized StaticString.withUTF8Buffer<A> ((UnsafeBufferPointer<UInt8>) -> A) -> A + 342
9    libswiftCore.dylib                 0x0000000113318130 specialized _assertionFailure(StaticString, String, file : StaticString, line : UInt, flags : UInt32) -> Never + 144
10   libswiftCore.dylib                 0x0000000113210130 swift_unexpectedError_merged + 289
12   swift                              0x000000010bfb9b40 llvm::MCJIT::runFunction(llvm::Function*, llvm::ArrayRef<llvm::GenericValue>) + 655
13   swift                              0x000000010bfbf700 llvm::ExecutionEngine::runFunctionAsMain(llvm::Function*, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, char const* const*) + 707
14   swift                              0x000000010b4bbf30 swift::RunImmediately(swift::CompilerInstance&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, swift::IRGenOptions&, swift::SILOptions const&) + 3385
15   swift                              0x000000010b484ff0 swift::performFrontend(llvm::ArrayRef<char const*>, char const*, void*, swift::FrontendObserver*) + 50738
16   swift                              0x000000010b43ca10 main + 9052
17   libdyld.dylib                      0x00007fffc82e8234 start + 1
0  swift                    0x000000010e6714f7 PrintStackTraceSignalHandler(void*) + 39
1  swift                    0x000000010e6709a6 SignalHandler(int) + 646
2  libsystem_platform.dylib 0x00007fffc84f7b3a _sigtramp + 26
3  libsystem_platform.dylib 000000000000000000 _sigtramp + 934315232
4  libswiftCore.dylib       0x0000000113210251 swift_unexpectedError_merged + 289
5  libswiftCore.dylib       0x0000000111554a81 swift_unexpectedError_merged + 4264839505
6  swift                    0x000000010bfb9dcf llvm::MCJIT::runFunction(llvm::Function*, llvm::ArrayRef<llvm::GenericValue>) + 655
7  swift                    0x000000010bfbf9c3 llvm::ExecutionEngine::runFunctionAsMain(llvm::Function*, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, char const* const*) + 707
8  swift                    0x000000010b4bcc69 swift::RunImmediately(swift::CompilerInstance&, std::__1::vector<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> >, std::__1::allocator<std::__1::basic_string<char, std::__1::char_traits<char>, std::__1::allocator<char> > > > const&, swift::IRGenOptions&, swift::SILOptions const&) + 3385
9  swift                    0x000000010b491622 swift::performFrontend(llvm::ArrayRef<char const*>, char const*, void*, swift::FrontendObserver*) + 50738
10 swift                    0x000000010b43ed6c main + 9052
11 libdyld.dylib            0x00007fffc82e8235 start + 1
12 libdyld.dylib            0x0000000000000011 start + 936476125
Stack dump:
0.	Program arguments: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/swift -frontend -interpret /var/folders/ds/tl29rv6n75n1x9lf__8yxtt40000gn/T/TemporaryFile.oHP2YK.swift -target x86_64-apple-macosx10.10 -enable-objc-interop -sdk /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.12.sdk -I /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/lib/swift/pm -module-name main -lPackageDescription -- -fileno 6
Child process exited with signal
Error swiftserver```

**Comment by tunniclm** [comment]
Hello @lucianboboc, the project doesn't yet support Swift 3.1. This is being addressed in #182.

**Comment by tunniclm** [comment]
I need to check it, but I think our last dependency got updated since yesterday. So hopefully this will be done soon.

**Comment by tunniclm** [comment]
This should have been addressed by version 2.1.0. Please reopen if there are still problems.
